osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -161,6 +161,7 @@ protected override void InitialiseDefaults()
             SetDefault(OsuSetting.ScalingPositionY, 0.5f, 0f, 1f);
 
             SetDefault(OsuSetting.UIScale, 1f, 0.8f, 1.6f, 0.01f);
+            SetDefault(OsuSetting.GlowStrength, 0.15f, 0.10f, 1.0f, 0.01f);
```

Comment:
Rename to `KiaiGlowStrength`

Reply:
Renamed to KiaiFlash in all places as suggested below.